adam/perception/perception_graph.py

```diff
@@ -538,7 +543,7 @@ def matches(
         )
         got_a_match = False
         for mapping in matching.subgraph_isomorphisms_iter(
-            debug=debug_mapping_sink is not None
+            debug=(debug_mapping_sink is not None) is not None
```

Comment:
I am confused by this line?

Reply:
Thanks for catching! Must be the Pycharm autocorrect. I'm pushing the fix now.